test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/ProjectNativeUtils.java

```diff
@@ -0,0 +1,58 @@
+package org.jboss.tools.openshift.reddeer.utils;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.osgi.util.NLS;
+import org.jboss.reddeer.common.logging.Logger;
+import org.jboss.reddeer.common.wait.WaitUntil;
+import org.jboss.tools.openshift.core.connection.Connection;
+import org.jboss.tools.openshift.reddeer.condition.OpenShiftProjectExists;
+import org.jboss.tools.openshift.reddeer.requirement.OpenShiftResourceUtils;
+
+import com.openshift.restclient.ResourceKind;
+import com.openshift.restclient.model.IProject;
+import com.openshift.restclient.model.project.IProjectRequest;
+
+public class ProjectNativeUtils {
```

Comment:
I see the added value of extracting this away 
I find this class name confusing. Out of the blue I would have thought that these are utils for Eclipse "native" projects, aka workspace projects. Dont you think so?
My pick would be to either move these 2 methods into the existing OpenShiftResourceUtils or have the class called OpenShiftProjectUtils (which would also match the other naming scheme of the general purpose resource utils class OpenShiftResourceUtils). Dont you think so?